validator/src/cli.rs

```diff
@@ -146,40 +137,6 @@ fn deprecated_arguments() -> Vec<DeprecatedArg> {
             .value_name("NUMBER"),
         usage_warning: "There is no more startup background accounts hash calculation",
     );
-    add_arg!(
-        // deprecated in v3.0.0
-        Arg::with_name("accounts_db_read_cache_limit_mb")
-            .long("accounts-db-read-cache-limit-mb")
-            .value_name("MAX | LOW,HIGH")
-            .takes_value(true)
-            .min_values(1)
-            .max_values(2)
-            .multiple(false)
-            .require_delimiter(true)
-            .help("How large the read cache for account data can become, in mebibytes")
-            .long_help(
-                "How large the read cache for account data can become, in mebibytes. \
-                 If given a single value, it will be the maximum size for the cache. \
-                 If given a pair of values, they will be the low and high watermarks \
-                 for the cache. When the cache exceeds the high watermark, entries will \
-                 be evicted until the size reaches the low watermark."
-            )
-            .hidden(hidden_unless_forced())
-            .conflicts_with("accounts_db_read_cache_limit"),
-            replaced_by: "accounts-db-read-cache-limit",
-    );
-    add_arg!(
-        // deprecated in v3.0.0
-        Arg::with_name("accounts_hash_cache_path")
-            .long("accounts-hash-cache-path")
-            .value_name("PATH")
-            .takes_value(true)
-            .help(
-                "Use PATH as accounts hash cache location \
-                 [default: <LEDGER>/accounts_hash_cache]",
-            ),
-            usage_warning: "The accounts hash cache is obsolete",
-    );
     add_arg!(Arg::with_name("disable_accounts_disk_index")
         // (actually) deprecated in v3.1.0
```

Comment:
`--disable-accounts-disk-index` was deprecated in v3.1.0. Can we remove it now, or do we need to wait until v4.1?